Resources/Locale/en-US/prototypes/roles/antags.ftl

```diff
@@ -21,3 +21,6 @@ roles-antag-nuclear-operative-agent-objective = Like default operative, the team
 
 roles-antag-nuclear-operative-name = Nuclear operative
 roles-antag-nuclear-operative-objective = Find the nuke disk and blow up the station.
+
+roles-antag-space-ninja-name = Space Ninja
+roles-antag-space-ninja-objective = Energy sword everything, nom on electrical wires.
```

Comment:
It seems your goals are more subtle now? Perhaps update the objective?

Reply:
ye it was just homage to the wiki

Reply:
I do love those electrical wires :D